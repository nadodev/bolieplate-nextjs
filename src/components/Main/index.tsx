const Main = () => (
  <div>
    <h1>react avancado</h1>
  </div>
)
export default Main
