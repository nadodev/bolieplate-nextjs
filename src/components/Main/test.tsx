import { render, screen } from '@testing-library/react'
import Main from '.'

describe('Main', () => {
  it('should render correctly', () => {
    render(<Main />)
    expect(
      screen.getByRole('heading', { name: /react avancado/i })
    ).toBeInTheDocument()
  })
})
