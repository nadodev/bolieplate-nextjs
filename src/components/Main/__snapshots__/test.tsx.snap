// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
.c0 {
  background-color: #06092b;
  color: #fff;
  width: 100%;
  height: 100%;
}

.c1 {
  max-width: 1200px;
  margin: 0 auto;
  width: 100%;
  text-align: center;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  width: 25rem;
  margin-bottom: 2rem;
}

.c3 {
  font-size: 2.5rem;
  margin-bottom: 2rem;
}

.c4 {
  font-size: 2rem;
  font-weight: bold;
  max-width: 30rem;
}

.c5 {
  margin-top: 3rem;
  width: min(30rem,100%);
}

<main
  class="c0"
>
  <div
    class="c1"
  >
    <img
      alt="logotipo, Imagem de um átomo, e react avançado escrito ao lado"
      class="c2"
      src="/logo.svg"
    />
    <h1
      class="c3"
    >
      react avancado
    </h1>
    <h2
      class="c4"
    >
      Typescript, ReactJs e Styled Components
    </h2>
    <img
      alt="Um homen sentado na frente do computador"
      class="c5"
      src="/hero-illustration.svg"
    />
  </div>
</main>
`
